src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -61,33 +61,42 @@ def transform_gene_info(
     druggability = druggability[useful_columns]
 
     target_list = nest_fields(
-        df=target_list, grouping="ensembl_gene_id", new_column="target_nominations", drop_columns=["ensembl_gene_id"]
+        df=target_list,
+        grouping="ensembl_gene_id",
+        new_column="target_nominations",
+        drop_columns=["ensembl_gene_id"],
```

Comment:
Note: changes like this are from the `black` formatter, I didn't change anything functional.